functions/interface.php

```diff
@@ -68,7 +68,7 @@ function hmbkp_admin_notices() {
 
 	<?php if ( ! empty( $notices['backup_errors'] ) ) : ?>
 
-		<div id="hmbkp-warning" class="error fade">
+		<div id="hmbkp-warning-backup" class="error  notice is-dismissible">
```

Comment:
It looks like this is introducing dismissable notices, although I guess they don't dismiss permanently? Anyhow, could you update ticket to reflect these additional changes so we have history